doc/htconfig.md

```diff
@@ -1,14 +1,14 @@
 Config values that can only be set in .htconfig.php
 ===================================================
 
-There are some config values that haven't found their way into the administration page. This has several reasons. Maybe they are part of a 
-current development that isn't considered stable and will be added later in the administration page when it is considered safe. Or it triggers 
+There are some config values that haven't found their way into the administration page. This has several reasons. Maybe they are part of a
+current development that isn't considered stable and will be added later in the administration page when it is considered safe. Or it triggers
```

Comment:
Yes it is usually one sentence per line. In some cases there were problems with that rules due to the Markdown syntax. IIRC. So have a look at `/help` after changing things.

It helps a lot when identifying too long sentences in the documentation, which might be correct, but unreadable due the sheer length, which holds especially true in cases that English is not the mother tongue of the author and the sentence would make more sense in the mother tongue of the author, and it keeps the diffs short.